i18n/fr.yaml

```diff
@@ -5,74 +5,62 @@ twitter_description:
 title:
   other: "COVID Shield"
 intro:
-  other: "COVID Shield is a free exposure notification solution built
-    with privacy as its top priority. It was built by a group of
-    volunteers in order to help Canadians and the rest of the world safely
-    return to work."
+  other: "COVID Shield est une solution de notification d'exposition libre d'accès, conçue dans le respect absolu de la confidentialité. Elle a été créée par un groupe de bénévoles dans le but d'aider les Canadiens et le reste du monde à reprendre le travail dans de bonnes conditions de sécurité."
```

Comment:
We want to use "notification d'exposition **à la COVID-19**" as we did in the app so I suggest:
`COVID Shield est une solution de notification d'exposition à la COVID-19 libre d'accès, conçue dans le respect absolu de la confidentialité. Elle a été créée par un groupe de bénévoles dans le but d'aider les Canadiens et le reste du monde à reprendre le travail dans de bonnes conditions de sécurité.`